src/NovelImageRect.cpp

```diff
@@ -130,7 +131,7 @@ void NovelImageRect::configureObjectBuffers() {
   SDL_Surface* surface = IMG_Load(_imageDir.c_str());
 
   if (surface == nullptr) {
-    std::cerr << "ERROR: File load returned a null pointer! Cannot load texture!" << std::endl;
+    _console.log("File load returned a null pointer! Cannot load texture!", LogLevel::ERR);
```

Comment:
We should make this error match the error for shader loading.

Reply:
Done (for now, may change depending on how we handle Tanner's question on `ImageRect` handling the logging service).